packages/api-contracts/ai-scan-api.yaml

```diff
@@ -349,17 +359,18 @@ components:
                 code:
                     type: string
                     enum:
-                        - InternalError
```

Comment:
Is InternalError actually going away? I did not see which change in the PR would do this but it would be a breaking change for clients.

Reply:
It is service OpenAPI specification file I've updated. Just added missing enum values to have parity with code.